### ISSUE TITLE
Build failure of version 0.4.1

### ISSUE THREAD

**Issue by mmmm1998** [Pure-D/serve-d#65]
System:
````bash
[13:25:59]$ lsb_release -a
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 18.04.3 LTS
Release:        18.04
Codename:       bionic
````
D lang and dub versions
````bash
[13:27:41]$ dmd --version
DMD64 D Compiler v2.087.1
Copyright (C) 1999-2019 by The D Language Foundation, All Rights Reserved written by Walter Bright
[13:27:46]$ dub --version
DUB version 1.16.0, built on Aug  4 2019
````
Error
````
.dub/packages/serve-d-0.4.1/serve-d/source/served/extension.d(281,10): Error: no property isAbsolute for type string
````

Full Build log (with error)
````
[13:23:53]$ dub build serve-d
Building package serve-d in /home/mmmm1998/.dub/packages/serve-d-0.4.1/serve-d/
Running pre-generate commands for dfmt...
Running pre-generate commands for dscanner...
Performing "debug" build using /usr/bin/dmd for x86_64.
diet-complete 0.0.3: target for configuration "library" is up to date.
eventsystem 1.2.0: target for configuration "library" is up to date.
stdx-allocator 2.77.5: target for configuration "library" is up to date.
libdparse 0.10.13: target for configuration "library" is up to date.
libddoc 0.5.1: target for configuration "lib" is up to date.
dunit 1.0.15: target for configuration "library" is up to date.
painlesstraits 0.3.0: target for configuration "unittest" is up to date.
painlessjson 1.3.9: target for configuration "library" is up to date.
dfmt 0.9.0: target for configuration "library" is up to date.
emsi_containers 0.8.0-alpha.16: target for configuration "library" is up to date.
dsymbol 0.5.7: target for configuration "library" is up to date.
inifiled 1.3.1: target for configuration "library-quiet" is up to date.
dscanner 0.6.0: target for configuration "library" is up to date.
dub 1.12.1: target for configuration "library" is up to date.
isfreedesktop 0.1.1: target for configuration "library" is up to date.
xdgpaths 0.2.5: target for configuration "library" is up to date.
standardpaths 0.8.1: target for configuration "default" is up to date.
workspace-d 3.3.0: target for configuration "library" is up to date.
serve-d 0.4.1: building configuration "application"...
.dub/packages/serve-d-0.4.1/serve-d/source/served/extension.d(18,8): Deprecation: alias `std.json.JSON_TYPE` is deprecated - Use JSONType
.dub/packages/serve-d-0.4.1/serve-d/source/served/extension.d(18,8): Deprecation: alias `std.json.JSON_TYPE` is deprecated - Use JSONType
.dub/packages/serve-d-0.4.1/serve-d/source/served/commands/code_actions.d(21,8): Deprecation: alias `std.json.JSON_TYPE` is deprecated - Use JSONType
.dub/packages/serve-d-0.4.1/serve-d/source/served/commands/code_actions.d(21,8): Deprecation: alias `std.json.JSON_TYPE` is deprecated - Use JSONType
.dub/packages/serve-d-0.4.1/serve-d/source/served/commands/code_lens.d(17,8): Deprecation: alias `std.json.JSON_TYPE` is deprecated - Use JSONType
.dub/packages/serve-d-0.4.1/serve-d/source/served/commands/code_lens.d(17,8): Deprecation: alias `std.json.JSON_TYPE` is deprecated - Use JSONType
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: alias `std.json.JSON_TYPE` is deprecated - Use JSONType
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: enum member `std.json.JSONType.NULL` is deprecated - Use .null_
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: alias `std.json.JSON_TYPE` is deprecated - Use JSONType
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: enum member `std.json.JSONType.NULL` is deprecated - Use .null_
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: alias `std.json.JSON_TYPE` is deprecated - Use JSONType
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: enum member `std.json.JSONType.NULL` is deprecated - Use .null_
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: alias `std.json.JSON_TYPE` is deprecated - Use JSONType
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: enum member `std.json.JSONType.NULL` is deprecated - Use .null_
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: alias `std.json.JSON_TYPE` is deprecated - Use JSONType
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: enum member `std.json.JSONType.NULL` is deprecated - Use .null_
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: alias `std.json.JSON_TYPE` is deprecated - Use JSONType
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: enum member `std.json.JSONType.NULL` is deprecated - Use .null_
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: alias `std.json.JSON_TYPE` is deprecated - Use JSONType
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: enum member `std.json.JSONType.NULL` is deprecated - Use .null_
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: alias `std.json.JSON_TYPE` is deprecated - Use JSONType
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: enum member `std.json.JSONType.NULL` is deprecated - Use .null_
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: alias `std.json.JSON_TYPE` is deprecated - Use JSONType
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: enum member `std.json.JSONType.NULL` is deprecated - Use .null_
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: alias `std.json.JSON_TYPE` is deprecated - Use JSONType
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: enum member `std.json.JSONType.NULL` is deprecated - Use .null_
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: alias `std.json.JSON_TYPE` is deprecated - Use JSONType
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: enum member `std.json.JSONType.NULL` is deprecated - Use .null_
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: alias `std.json.JSON_TYPE` is deprecated - Use JSONType
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: enum member `std.json.JSONType.NULL` is deprecated - Use .null_
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: alias `std.json.JSON_TYPE` is deprecated - Use JSONType
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: enum member `std.json.JSONType.NULL` is deprecated - Use .null_
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: alias `std.json.JSON_TYPE` is deprecated - Use JSONType
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: enum member `std.json.JSONType.NULL` is deprecated - Use .null_
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: alias `std.json.JSON_TYPE` is deprecated - Use JSONType
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: enum member `std.json.JSONType.NULL` is deprecated - Use .null_
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: alias `std.json.JSON_TYPE` is deprecated - Use JSONType
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: enum member `std.json.JSONType.NULL` is deprecated - Use .null_
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: alias `std.json.JSON_TYPE` is deprecated - Use JSONType
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: enum member `std.json.JSONType.NULL` is deprecated - Use .null_
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: alias `std.json.JSON_TYPE` is deprecated - Use JSONType
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: enum member `std.json.JSONType.NULL` is deprecated - Use .null_
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: alias `std.json.JSON_TYPE` is deprecated - Use JSONType
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: enum member `std.json.JSONType.NULL` is deprecated - Use .null_
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: alias `std.json.JSON_TYPE` is deprecated - Use JSONType
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: enum member `std.json.JSONType.NULL` is deprecated - Use .null_
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: alias `std.json.JSON_TYPE` is deprecated - Use JSONType
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: enum member `std.json.JSONType.NULL` is deprecated - Use .null_
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: alias `std.json.JSON_TYPE` is deprecated - Use JSONType
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: enum member `std.json.JSONType.NULL` is deprecated - Use .null_
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: alias `std.json.JSON_TYPE` is deprecated - Use JSONType
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: enum member `std.json.JSONType.NULL` is deprecated - Use .null_
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: alias `std.json.JSON_TYPE` is deprecated - Use JSONType
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: enum member `std.json.JSONType.NULL` is deprecated - Use .null_
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: alias `std.json.JSON_TYPE` is deprecated - Use JSONType
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: enum member `std.json.JSONType.NULL` is deprecated - Use .null_
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: alias `std.json.JSON_TYPE` is deprecated - Use JSONType
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: enum member `std.json.JSONType.NULL` is deprecated - Use .null_
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: alias `std.json.JSON_TYPE` is deprecated - Use JSONType
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: enum member `std.json.JSONType.NULL` is deprecated - Use .null_
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: alias `std.json.JSON_TYPE` is deprecated - Use JSONType
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: enum member `std.json.JSONType.NULL` is deprecated - Use .null_
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: alias `std.json.JSON_TYPE` is deprecated - Use JSONType
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: enum member `std.json.JSONType.NULL` is deprecated - Use .null_
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: alias `std.json.JSON_TYPE` is deprecated - Use JSONType
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: enum member `std.json.JSONType.NULL` is deprecated - Use .null_
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: alias `std.json.JSON_TYPE` is deprecated - Use JSONType
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: enum member `std.json.JSONType.NULL` is deprecated - Use .null_
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: alias `std.json.JSON_TYPE` is deprecated - Use JSONType
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: enum member `std.json.JSONType.NULL` is deprecated - Use .null_
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: alias `std.json.JSON_TYPE` is deprecated - Use JSONType
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: enum member `std.json.JSONType.NULL` is deprecated - Use .null_
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: alias `std.json.JSON_TYPE` is deprecated - Use JSONType
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: enum member `std.json.JSONType.NULL` is deprecated - Use .null_
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: alias `std.json.JSON_TYPE` is deprecated - Use JSONType
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: enum member `std.json.JSONType.NULL` is deprecated - Use .null_
.dub/packages/serve-d-0.4.1/serve-d/source/served/commands/code_actions.d(147,41): Deprecation: enum member `std.json.JSONType.STRING` is deprecated - Use .string
.dub/packages/serve-d-0.4.1/serve-d/source/served/commands/code_lens.d(44,24): Deprecation: enum member `std.json.JSONType.OBJECT` is deprecated - Use .object
.dub/packages/serve-d-0.4.1/serve-d/source/served/commands/code_lens.d(55,30): Deprecation: enum member `std.json.JSONType.STRING` is deprecated - Use .string
.dub/packages/serve-d-0.4.1/serve-d/source/served/commands/code_lens.d(58,36): Deprecation: enum member `std.json.JSONType.STRING` is deprecated - Use .string
.dub/packages/serve-d-0.4.1/serve-d/source/served/commands/code_lens.d(61,40): Deprecation: enum member `std.json.JSONType.STRING` is deprecated - Use .string
.dub/packages/workspace-d-3.3.0/workspace-d/source/workspaced/api.d(312,20): Deprecation: alias `std.json.JSON_TYPE` is deprecated - Use JSONType
.dub/packages/workspace-d-3.3.0/workspace-d/source/workspaced/api.d(312,20): Deprecation: enum member `std.json.JSONType.OBJECT` is deprecated - Use .object
.dub/packages/serve-d-0.4.1/serve-d/source/served/extension.d(207,26): Deprecation: enum member `std.json.JSONType.ARRAY` is deprecated - Use .array
.dub/packages/serve-d-0.4.1/serve-d/source/served/extension.d(253,26): Deprecation: enum member `std.json.JSONType.ARRAY` is deprecated - Use .array
.dub/packages/serve-d-0.4.1/serve-d/source/served/extension.d(281,10): Error: no property isAbsolute for type string
.dub/packages/serve-d-0.4.1/serve-d/source/served/extension.d(281,48): Deprecation: function `served.types.workspaceRoot` is deprecated
/usr/include/dmd/phobos/std/algorithm/iteration.d(499,16):        instantiated from here: MapResult!(__lambda2, string[])
.dub/packages/serve-d-0.4.1/serve-d/source/served/extension.d(280,82):        instantiated from here: map!(string[])
.dub/packages/workspace-d-3.3.0/workspace-d/source/workspaced/api.d(312,20): Deprecation: alias `std.json.JSON_TYPE` is deprecated - Use JSONType
.dub/packages/workspace-d-3.3.0/workspace-d/source/workspaced/api.d(312,20): Deprecation: enum member `std.json.JSONType.OBJECT` is deprecated - Use .object
.dub/packages/serve-d-0.4.1/serve-d/source/served/extension.d(570,27): Deprecation: enum member `std.json.JSONType.STRING` is deprecated - Use .string
.dub/packages/workspace-d-3.3.0/workspace-d/source/workspaced/api.d(312,20): Deprecation: alias `std.json.JSON_TYPE` is deprecated - Use JSONType
.dub/packages/workspace-d-3.3.0/workspace-d/source/workspaced/api.d(312,20): Deprecation: enum member `std.json.JSONType.OBJECT` is deprecated - Use .object
.dub/packages/serve-d-0.4.1/serve-d/source/served/jsonrpc.d(43,26): Deprecation: alias `std.json.JSON_TYPE` is deprecated - Use JSONType
.dub/packages/serve-d-0.4.1/serve-d/source/served/jsonrpc.d(43,26): Deprecation: enum member `std.json.JSONType.NULL` is deprecated - Use .null_
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: alias `std.json.JSON_TYPE` is deprecated - Use JSONType
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: enum member `std.json.JSONType.NULL` is deprecated - Use .null_
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: alias `std.json.JSON_TYPE` is deprecated - Use JSONType
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: enum member `std.json.JSONType.NULL` is deprecated - Use .null_
.dub/packages/serve-d-0.4.1/serve-d/source/served/protocol.d(171,19): Deprecation: alias `std.json.JSON_TYPE` is deprecated - Use JSONType
.dub/packages/serve-d-0.4.1/serve-d/source/served/protocol.d(171,19): Deprecation: enum member `std.json.JSONType.STRING` is deprecated - Use .string
.dub/packages/serve-d-0.4.1/serve-d/source/served/protocol.d(176,24): Deprecation: alias `std.json.JSON_TYPE` is deprecated - Use JSONType
.dub/packages/serve-d-0.4.1/serve-d/source/served/protocol.d(176,24): Deprecation: enum member `std.json.JSONType.INTEGER` is deprecated - Use .integer
.dub/packages/serve-d-0.4.1/serve-d/source/served/protocol.d(1070,19): Deprecation: alias `std.json.JSON_TYPE` is deprecated - Use JSONType
.dub/packages/serve-d-0.4.1/serve-d/source/served/protocol.d(1070,19): Deprecation: enum member `std.json.JSONType.STRING` is deprecated - Use .string
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: alias `std.json.JSON_TYPE` is deprecated - Use JSONType
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: enum member `std.json.JSONType.NULL` is deprecated - Use .null_
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: alias `std.json.JSON_TYPE` is deprecated - Use JSONType
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: enum member `std.json.JSONType.NULL` is deprecated - Use .null_
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: alias `std.json.JSON_TYPE` is deprecated - Use JSONType
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: enum member `std.json.JSONType.NULL` is deprecated - Use .null_
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: alias `std.json.JSON_TYPE` is deprecated - Use JSONType
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: enum member `std.json.JSONType.NULL` is deprecated - Use .null_
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: alias `std.json.JSON_TYPE` is deprecated - Use JSONType
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: enum member `std.json.JSONType.NULL` is deprecated - Use .null_
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: alias `std.json.JSON_TYPE` is deprecated - Use JSONType
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: enum member `std.json.JSONType.NULL` is deprecated - Use .null_
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: alias `std.json.JSON_TYPE` is deprecated - Use JSONType
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: enum member `std.json.JSONType.NULL` is deprecated - Use .null_
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: alias `std.json.JSON_TYPE` is deprecated - Use JSONType
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: enum member `std.json.JSONType.NULL` is deprecated - Use .null_
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: alias `std.json.JSON_TYPE` is deprecated - Use JSONType
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: enum member `std.json.JSONType.NULL` is deprecated - Use .null_
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: alias `std.json.JSON_TYPE` is deprecated - Use JSONType
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: enum member `std.json.JSONType.NULL` is deprecated - Use .null_
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: alias `std.json.JSON_TYPE` is deprecated - Use JSONType
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: enum member `std.json.JSONType.NULL` is deprecated - Use .null_
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: alias `std.json.JSON_TYPE` is deprecated - Use JSONType
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: enum member `std.json.JSONType.NULL` is deprecated - Use .null_
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: alias `std.json.JSON_TYPE` is deprecated - Use JSONType
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: enum member `std.json.JSONType.NULL` is deprecated - Use .null_
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: alias `std.json.JSON_TYPE` is deprecated - Use JSONType
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: enum member `std.json.JSONType.NULL` is deprecated - Use .null_
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: alias `std.json.JSON_TYPE` is deprecated - Use JSONType
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: enum member `std.json.JSONType.NULL` is deprecated - Use .null_
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: alias `std.json.JSON_TYPE` is deprecated - Use JSONType
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: enum member `std.json.JSONType.NULL` is deprecated - Use .null_
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: alias `std.json.JSON_TYPE` is deprecated - Use JSONType
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: enum member `std.json.JSONType.NULL` is deprecated - Use .null_
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: alias `std.json.JSON_TYPE` is deprecated - Use JSONType
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: enum member `std.json.JSONType.NULL` is deprecated - Use .null_
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: alias `std.json.JSON_TYPE` is deprecated - Use JSONType
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: enum member `std.json.JSONType.NULL` is deprecated - Use .null_
.dub/packages/serve-d-0.4.1/serve-d/source/served/protocol.d(151,19): Deprecation: alias `std.json.JSON_TYPE` is deprecated - Use JSONType
.dub/packages/serve-d-0.4.1/serve-d/source/served/protocol.d(151,19): Deprecation: enum member `std.json.JSONType.ARRAY` is deprecated - Use .array
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: alias `std.json.JSON_TYPE` is deprecated - Use JSONType
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: enum member `std.json.JSONType.NULL` is deprecated - Use .null_
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: alias `std.json.JSON_TYPE` is deprecated - Use JSONType
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: enum member `std.json.JSONType.NULL` is deprecated - Use .null_
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: alias `std.json.JSON_TYPE` is deprecated - Use JSONType
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: enum member `std.json.JSONType.NULL` is deprecated - Use .null_
.dub/packages/serve-d-0.4.1/serve-d/source/served/protocol.d(151,19): Deprecation: alias `std.json.JSON_TYPE` is deprecated - Use JSONType
.dub/packages/serve-d-0.4.1/serve-d/source/served/protocol.d(151,19): Deprecation: enum member `std.json.JSONType.ARRAY` is deprecated - Use .array
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: alias `std.json.JSON_TYPE` is deprecated - Use JSONType
.dub/packages/painlessjson-1.3.9/painlessjson/source/painlessjson/painlessjson.d(629,26): Deprecation: enum member `std.json.JSONType.NULL` is deprecated - Use .null_
.dub/packages/serve-d-0.4.1/serve-d/source/served/types.d(122,17): Deprecation: alias `std.json.JSON_TYPE` is deprecated - Use JSONType
.dub/packages/serve-d-0.4.1/serve-d/source/served/types.d(122,17): Deprecation: enum member `std.json.JSONType.ARRAY` is deprecated - Use .array
.dub/packages/serve-d-0.4.1/serve-d/source/served/types.d(126,18): Deprecation: alias `std.json.JSON_TYPE` is deprecated - Use JSONType
.dub/packages/serve-d-0.4.1/serve-d/source/served/types.d(126,18): Deprecation: enum member `std.json.JSONType.STRING` is deprecated - Use .string
.dub/packages/serve-d-0.4.1/serve-d/source/served/types.d(454,20): Deprecation: alias `std.json.JSON_TYPE` is deprecated - Use JSONType
.dub/packages/serve-d-0.4.1/serve-d/source/served/types.d(454,20): Deprecation: enum member `std.json.JSONType.UINTEGER` is deprecated - Use .uinteger
/usr/bin/dmd failed with exit code 1.
````

**Comment by WebFreak001** [comment]
downgrade compiler or update serve-d (if you compile via code-d, set `"d.betaStream": true`)

**Comment by mmmm1998** [comment]
2.087.1 version not beta, but okay, which compiler version supported by serve-d?

**Comment by WebFreak001** [comment]
0.4.1 and before is only supported with dmd below 2.087

if you add `"d.betaStream": true` to your user settings, code-d will grab the latest serve-d commit (master) and compile with your 2.087.1 dmd. The latest dmd compiler silently broke compilation for most versions so it's only fixed in the most recent commits which you get by d.betaStream

**Comment by WebFreak001** [comment]
fixed with 0.5.1
